cuhttp/server.go

```diff
@@ -160,20 +153,35 @@ func StartHTTPServer(ctx context.Context, d *deps.Deps) error {
 		Addr:              cfg.ListenAddress,
 		Handler:           libp2pConnMiddleware(loggingMiddleware(compressionMw(chiRouter))), // Attach middlewares
 		ReadTimeout:       cfg.ReadTimeout,
-		WriteTimeout:      cfg.WriteTimeout,
+		WriteTimeout:      time.Hour * 2,
```

Comment:
So WriteTimeout generally isn't useful for mitigating much, unless you're trying to mitigate successful retrievals.

Later we could bring a much more advanced write rate enforcer from ribs so that /piece uploads have reasonable bounds, but for now setting a low write timeout guarantees that retrievals will fail

Ref https://github.com/FilOzone/ribs/blob/main/ributil/minratewriter.go